### PR TITLE
[RA2 Ch02]: Changing NFVI to Cloud Infrastructure attribute name changes

### PR DESCRIPTION
**Pull request by CsatariGergely**
This pr should be merged only after #1369, #1372 and #1413

### CODE REVIEW

File: doc/ref_arch/kubernetes/chapters/chapter02.md [hunk 1]
```diff
@@ -95,18 +95,18 @@ From Reference Model section [4.2.5 Instance Capabilities Mapping](../../../ref_
...
-| e.nfvi.res.cap.004 | # Connection Points | 6 | Y |
-| e.nfvi.res.cap.005 | Total instance (persistent) storage (GB) | Up to 16TB<sup>2</sup> | Y |
-| e.nfvi.per.cap.001 | CPU pinning support | | Y |
-| e.nfvi.per.cap.002 | NUMA support | | Y |
-| e.nfvi.per.cap.003 | IPSec Acceleration | | Yes (if offered) see section [2.2.1 Network Acceleration Extensions](#2.2.1) |
-| e.nfvi.per.cap.004 | Crypto Acceleration | | Yes (if offered) see section [2.2.1 Network Acceleration Extensions](#2.2.1) |
-| e.nfvi.per.cap.005 | Transcoding Acceleration | | Yes (if offered) see section [2.2.1 Network Acceleration Extensions](#2.2.1) |
-| e.nfvi.per.cap.006 | Programmable Acceleration | | Yes (if offered) see section [2.2.1 Network Acceleration Extensions](#2.2.1) |
-| e.nfvi.per.cap.007 | Enhanced Cache Management | | X (if offered) |
+| e.res.cap.001 | Max number of vCPU that can be assigned to a single pod by the NFVI | at least 16<sup>1)</sup> | Y |
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter02.md]: e.res.cap.001, e.res.cap.002 and e.res.cap.003 have NFVI in their descriptions that need to be changed

Comment (pgoyal01) [reply doc/ref_arch/kubernetes/chapters/chapter02.md]: There are other places too -- sent in an email

File: doc/ref_arch/kubernetes/chapters/chapter02.md [hunk 2]
```diff
@@ -95,27 +95,35 @@ From Reference Model section [4.2.5 Instance Capabilities Mapping](../../../ref_
...
+| i.cap.017 | Connection point QoS | | Y |
+| i.cap.018 | Huge page support |  | Y |
+| i.pm.001 | Monitor host CPU usage |  | Y |
+| i.pm.002 | Monitor virtual compute CPU usage | | Y |
+| i.pm.003 | Monitor host CPU utilization | | Y |
+| i.pm.004 | Monitor virtual compute CPU utilization | | Y |
+| i.pm.005 | Measurement of external storage IOPs | | ? |
+| i.pm.006 | Measurement of external storage throughput | | ? |
+| i.pm.007 | Available external storage capacity | | ? |
 | nfvi.com.cfg.002 | NUMA awareness | | Y |
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter02.md:127]: @CsatariGergely some of these "nfvi.com.cfg.xxx" items are also listed in your "e.cap.xxx" lists. For example, NUMA -- e.cap.007 and nfvi.com.cfg.002

Comment (CsatariGergely) [reply doc/ref_arch/kubernetes/chapters/chapter02.md]: Yes, these attributes are duplicated in the RM. They are required in Chapter 4 or the RM as exposed capabilities of the Cloud Infrastructure, like `e.cap.007` and in Chapter 5 of the RM as part of the Cloud Infrastructure Software profile, like `infra.com.cfg.002`.
I think this chapter should only copy what is there in the RM and if we would like to get rid of these duplication we should first remove them from the RM. @kedmison, @pgoyal01 what do you think?

Comment (pgoyal01) [reply doc/ref_arch/kubernetes/chapters/chapter02.md]: There is an issue raised for RM to do so.

**Pull request merged**